src/render-style-transfer.py

```diff
@@ -256,8 +254,11 @@ def main(keep_logs):
     # mac paths for shared directory
     # trainset = RenderStyleTransferDataset(root_dir="/Volumes/aics/animated-cell/Dan/renderstyletransfer/training_data", train=True)
     # testset = RenderStyleTransferDataset(root_dir="/Volumes/aics/animated-cell/Dan/renderstyletransfer/training_data", train=False)
-    trainset = PrecomputedStyleTransferDataset(cache_file="cached/dataset.json", train=True)
-    testset = PrecomputedStyleTransferDataset(cache_file="cached/dataset.json", train=False)
+    data_dir = "/Volumes/aics/animated-cell/Dan/renderstyletransfer/training_data"
+
+    trainset = StyleTransferDataset(data_dir, cache_setting="load", train=True)
```

Comment:
is data_dir ignored if you set cache_setting to "load" ?

Reply:
yeah